NSAttributedString+Markdown.m

```diff
@@ -541,6 +571,74 @@ static void updateAttributedString(NSMutableAttributedString *result, NSString *
 							NSCAssert(NO, @"Not implemented");
 #endif
 							break;
+                        case MarkdownSpanListItem: {
+#if ALLOW_BULLETED_LISTS
+                            // check characters prior to bullet marker for indent level (4 spaces per indent)
+                            NSUInteger spacesBeforeBullet = 0;
+                            if (beginRange.location > 0) {
+                                NSInteger position = beginRange.location - 1;
+                                NSRange checkCharacterRange = NSMakeRange(position, 1);
+                                while ([[scanString substringWithRange:checkCharacterRange] isEqualToString:@" "] && position > 0) {
+                                    position--;
+                                    spacesBeforeBullet++;
+                                    checkCharacterRange = NSMakeRange(position, 1);
+                                }
+                                // verify character before bullet is either start of string or new line, otherwise could be a standard dash-space (e.g., "This - here - shouldn't create bullets")
+                                if (position > 0 && ![[scanString substringWithRange:NSMakeRange(position, 1)] isEqualToString:@"\n"]) {
+                                    // skip
+                                    replaceMarkers = NO;
+                                    break;
+                                }
+                            }
+                            NSUInteger bulletLevel = spacesBeforeBullet / 4;
```

Comment:
Would be nice to define the `4` value as a constant at the top of the file

Reply:
Agreed